install-hydra.sh

```diff
@@ -1,26 +1,28 @@
 #!/usr/bin/env bash
-HYDRA_LINK_PATH=/usr/local/bin/hydra
-if ! docker --version ; then
-    echo "Docker not installed!!!"
-    curl -fsSL get.docker.com --retry 5 --retry-max-time 300  -o get-docker.sh
-    sh get-docker.sh
-    # Docker post install, adds ability to run Docker as current user
-    groupadd docker || true
-    usermod -aG docker ${USER} || true
-    echo "==================================================================================================="
-    echo "!!!! In order to run Docker as a current user, please logout from your current session !!!!"
-    echo "More info: https://docs.docker.com/install/linux/linux-postinstall/#manage-docker-as-a-non-root-user"
-    echo "==================================================================================================="
+HYDRA_LINK_PATH = /usr/local/bin/hydra
+if ! docker - -version
+then
+echo "Docker not installed!!!"
+curl - fsSL get.docker.com - -retry 5 - -retry-max-time 300 - o get-docker.sh
```

Comment:
The same here.

Reply:
thanks for finding it. I just blindly accepted autopep8 fixes. fixed.